pkg/riff/commands/function.go

```diff
@@ -29,7 +29,23 @@ func NewFunctionCommand(ctx context.Context, c *cli.Config) *cobra.Command {
 		Use:   "function",
 		Short: "functions built from source using function buildpacks",
 		Long: strings.TrimSpace(`
-<todo>
+Functions are a mechanism for converting language idiomatic units of logic into
```

Comment:
Not really.
```suggestion
Functions are idiomatic units of code which are converted into
```

Reply:
This line is talking about the Function resource, I think you're talking about the programming language concept.